modules/ROOT/pages/db/database-connector-examples.adoc

```diff
@@ -518,6 +518,50 @@ After execution, the resulting values of the `output` and `input-output` paramet
 
 For Database Connector version 1.4.0 or later, make sure that no whitespace is between the name of the stored procedure and the first parentheses.
 
+Database Connector supports the use of callable statements with the format ``{ call procedureName(:param1, :param2, ..., :paramN) }`` where parameters matches positionally, that is to say, `:paramN` matches the `Nth` parameter in the stored procedure declaration.
+
+Suppose you are using an Oracle database which you have initialized using with the following table and stored procedure:
```

Comment:
Clarify the "using" / "with" wording by choosing one that matches your intent:
Was the database initialized by using the table and procedure?
Or was the database initialized using the values in the table and the procedure?
Or was the database initialized to include the values in the table and the procedure?